nilearn/datasets/atlas.py

```diff
@@ -576,3 +576,73 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
         url=url,
         resume=resume,
         verbose=verbose)
+
+
+def fetch_atlas_aal(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the AAL for SPM 12 parcellation.
+
+    The provided images are based on the spatially normalized single-subject
+    high-resolution T1 volume provided by the Montreal Neurological Institute
+    (MNI)
+
+    Parameters
+    ----------
+    data_dir: string
+        directory where data should be downloaded and unpacked.
+
+    url: string
+        url of file to download.
+
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+
+        - "anat": anatomy image.
```

Comment:
We can change the name but in that case, we have to change `harvard_oxford` too to be homogeneous. No good naming comes to my mind we I can summon our naming specialist @banilo ;).